cypress/integration/tests/manage-germplasm/germplasm-details.feature

```diff
@@ -7,16 +7,18 @@ Feature: Germplasm Details
 
 @TestCaseKey=IBP-T3178
 @sanity-test
-    Scenario: Check pedigree tree
-        Given I am on the Germplasm Manager page
+    Scenario: Check pedigree 
```

Comment:
As discussed, create scenarios for clean and update install as for update install sanity test, we shouldn't be adding data

Reply:
Done